canon/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## [0.6.4] 2021-06-04
+
+### Changed
+- Change `Id::encoded_len` to a correct implementation
+- Change `Repr` to be deprecated
```

Comment:
Okay, now it make sense. However, `Deprecated` is a different section see: https://keepachangelog.com/en/1.0.0/